src/io/flutter/editor/WidgetIndentsHighlightingPass.java

```diff
@@ -347,7 +357,7 @@ public void paint(@NotNull Editor editor, @NotNull RangeHighlighter highlighter,
                 start.x + 2,
                 newY + lineHeight * 0.5,
                 //start.x + charWidth  * childIndent - padding,
-                widgetPoint.x - padding,
+                widgetLineStartX - padding,
```

Comment:
Also update the edge case where we draw a line backward (starts at line 364). A reasonable option would be to just omit drawing the line backward if widgetLineStartX != widgetPoint.x as the backwards lines are distracting so drawing them for a case where we aren't confident that something strange has happened would be a bad thing.

Reply:
I'm unclear on the backwards lines - the original situation would cause a loop to be drawn from the parent element to the beginning of the widget (before any outline update):
<img width="592" alt="Screen Shot 2020-05-01 at 9 38 23 AM" src="https://user-images.githubusercontent.com/6379305/80822584-e2361200-8b8f-11ea-9174-ff91db2d3242.png">
This seems okay? But is it confusing because maybe the loop should be drawn to before `child:`?

With this change, then there would be no backwards line in this case:
<img width="572" alt="Screen Shot 2020-05-01 at 9 42 27 AM" src="https://user-images.githubusercontent.com/6379305/80822680-18739180-8b90-11ea-98fa-b6c618d21224.png">

Reply:
some background: the backwards lines don't really provide much value as they look pretty bad so that is why we should stop showing them if there is a risk we are showing a misleading line. The main reasons we show them at all is because it is easy to show them and because that explains why we don't draw a line in cases where the user's code isn't indented nicely. 
in the case you are dealing with where the outline doesn't match the current source code, it is safer to just not show the line.

Reply:
the blue lines from the row to its children also look bad in this case so it is a good thing if we don't show them when given bad data like this. In this case they are on the wrong side of the lines for their parents so the guides are hard to read so distract more than add value.